cms/jinja2/component_overrides/header/_macro.njk

```diff
@@ -0,0 +1,224 @@
+{% macro onsHeaderNew(params) %}
```

Comment:
We dropped this component override in #212

Reply:
Thank you. If only we had somebody who knew the project inside out. This is removed in aa70fd2662ccae54e66d3fdc14986904b3858a61.